src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx

```diff
@@ -63,6 +67,18 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
               </TableRow>
             ))}
           </TableBody>
+          {index === tables.length - 1 && showFooter && (
+            <Footer isLight={isLight} isPar={isPar}>
```

Comment:
there is a `isPar` here too

Reply:
yes, its the same variable than is pass to component, its done